libraries/freertos_plus/standard/freertos_plus_tcp/include/FreeRTOS_IP_Private.h

```diff
@@ -53,6 +56,7 @@ typedef struct xNetworkAddressingParameters
 } NetworkAddressingParameters_t;
 
 extern BaseType_t xTCPWindowLoggingLevel;
+extern QueueHandle_t xNetworkEventQueue;
```

Comment:
Is `QueueHandle_t` accessible from this file? On compiling, I get an error for this.

Reply:
PC-lint urges programmers not to include unnecessary header files.
`semphr.h` is already included depending on this macro:
~~~c
#if( ipconfigSOCKET_HAS_USER_SEMAPHORE == 1 )
	#include "semphr.h"
#endif
~~~
I will change that to an unconditional:
~~~c
#include "semphr.h"
~~~

Reply:
Thanks